### ISSUE TITLE
Troubleshooting logs doesn't always contain stacktrace

### ISSUE THREAD

**Issue by flamber** [metabase/metabase#12851]
**Describe the bug**
Sometimes the Admin > Troubleshooting > Logs doesn't contain the stacktrace. This has been seen during debugging of sync and pulse issues.

**To Reproduce**
_When issue #8630 is fixed, this will need other steps to reproduce_
1. Add the Food Trucks SQLite database - Admin > Databases
2. Admin > Troubleshooting > Logs will include this:
```
[a804b461-958a-475c-bef9-9deee0c2705d] 2020-07-06T23:30:35+02:00 WARN metabase.driver.sql-jdbc.sync Don't know how to map column type '' to a Field base_type, falling back to :type/*.
[a804b461-958a-475c-bef9-9deee0c2705d] 2020-07-06T23:30:35+02:00 ERROR metabase.sync.util Error syncing Fields for Table 'Table 16 'Mobile_Food_Facility_Permit_fts''
[a804b461-958a-475c-bef9-9deee0c2705d] 2020-07-06T23:30:35+02:00 INFO metabase.sync.util FINISHED: step 'sync-fields' for sqlite Database 5 'SQLite Food Trucks' (167.9 ms)
```
3. Server console log will include this:
```
07-06 23:30:35 WARN sql-jdbc.sync :: Don't know how to map column type '' to a Field base_type, falling back to :type/*.
07-06 23:30:35 ERROR sync.util :: Error syncing Fields for Table 'Table 16 'Mobile_Food_Facility_Permit_fts''
clojure.lang.ExceptionInfo: Output of table-metadata does not match schema: 

	   {:fields #{{:database-type (not ("Non-blank string" ""))} {:database-type (not ("Non-blank string" ""))} {:database-type (not ("Non-blank string" ""))} {:database-type (not ("Non-blank string" ""))} {:database-type (not ("Non-blank string" ""))}}}  

 {:type :schema.core/error, :schema {:name (constrained Str "Non-blank string"), :schema (maybe (constrained Str "Non-blank string")), :fields #{{:name (constrained Str "Non-blank string"), :database-type (maybe (constrained Str "Non-blank string")), :base-type (pred "Valid field type"), #schema.core.OptionalKey{:k :special-type} (maybe (pred "Valid field type")), #schema.core.OptionalKey{:k :field-comment} (maybe (constrained Str "Non-blank string")), #schema.core.OptionalKey{:k :pk?} java.lang.Boolean, #schema.core.OptionalKey{:k :nested-fields} #{(recursive (var metabase.sync.interface/TableMetadataField))}, #schema.core.OptionalKey{:k :custom} {Any Any}}}, #schema.core.OptionalKey{:k :description} (maybe (constrained Str "Non-blank string"))}, :value {:name "Mobile_Food_Facility_Permit_fts", :schema nil, :fields #{{:name "Address", :database-type "", :base-type :type/*} {:name "FacilityType", :database-type "", :base-type :type/*} {:name "LocationDescription", :database-type "", :base-type :type/*} {:name "FoodItems", :database-type "", :base-type :type/*} {:name "Applicant", :database-type "", :base-type :type/*}}}, :error {:fields #{{:database-type (not ("Non-blank string" ""))} {:database-type (not ("Non-blank string" ""))} {:database-type (not ("Non-blank string" ""))} {:database-type (not ("Non-blank string" ""))} {:database-type (not ("Non-blank string" ""))}}}}
	at metabase.sync.fetch_metadata$fn__52594$table_metadata__52599.invoke(fetch_metadata.clj:15)
	at metabase.sync.sync_metadata.fields.fetch_metadata$fn__52916$db_metadata__52921$fn__52922.invoke(fetch_metadata.clj:82)
	at metabase.sync.sync_metadata.fields.fetch_metadata$fn__52916$db_metadata__52921.invoke(fetch_metadata.clj:78)
	at metabase.sync.sync_metadata.fields$fn__53419$sync_fields_for_table_BANG___53428$fn__53434$fn__53436.invoke(fields.clj:90)
	at metabase.sync.util$do_with_error_handling.invokeStatic(util.clj:149)
	at metabase.sync.util$do_with_error_handling.invoke(util.clj:142)
	at metabase.sync.sync_metadata.fields$fn__53419$sync_fields_for_table_BANG___53428$fn__53434.invoke(fields.clj:89)
	at metabase.sync.sync_metadata.fields$fn__53419$sync_fields_for_table_BANG___53428.invoke(fields.clj:83)
	at clojure.core$partial$fn__5839.invoke(core.clj:2624)
	at clojure.core$map$fn__5866.invoke(core.clj:2753)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:51)
	at clojure.lang.RT.seq(RT.java:535)
	at clojure.core$seq__5402.invokeStatic(core.clj:137)
	at clojure.core$filter$fn__5893.invoke(core.clj:2809)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:51)
	at clojure.lang.Cons.next(Cons.java:39)
	at clojure.lang.RT.boundedLength(RT.java:1792)
	at clojure.lang.RestFn.applyTo(RestFn.java:130)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at metabase.sync.sync_metadata.fields$fn__53464$sync_fields_BANG___53469$fn__53470.invoke(fields.clj:115)
	at metabase.sync.sync_metadata.fields$fn__53464$sync_fields_BANG___53469.invoke(fields.clj:107)
	at metabase.sync.util$fn__44568$run_step_with_metadata__44573$fn__44577$fn__44579.invoke(util.clj:345)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invokeStatic(util.clj:102)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invoke(util.clj:96)
	at metabase.sync.util$with_start_and_finish_debug_logging.invokeStatic(util.clj:119)
	at metabase.sync.util$with_start_and_finish_debug_logging.invoke(util.clj:116)
	at metabase.sync.util$fn__44568$run_step_with_metadata__44573$fn__44577.invoke(util.clj:342)
	at metabase.sync.util$fn__44568$run_step_with_metadata__44573.invoke(util.clj:337)
	at metabase.sync.util$fn__44760$run_sync_operation__44765$fn__44766$fn__44767.invoke(util.clj:430)
	at clojure.core$mapv$fn__8445.invoke(core.clj:6912)
	at clojure.lang.PersistentVector.reduce(PersistentVector.java:343)
	at clojure.core$reduce.invokeStatic(core.clj:6827)
	at clojure.core$mapv.invokeStatic(core.clj:6903)
	at clojure.core$mapv.invoke(core.clj:6903)
	at metabase.sync.util$fn__44760$run_sync_operation__44765$fn__44766.invoke(util.clj:430)
	at metabase.sync.util$fn__44760$run_sync_operation__44765.invoke(util.clj:424)
	at metabase.sync.sync_metadata$fn__54138$sync_db_metadata_BANG___54143$fn__54144$fn__54145.invoke(sync_metadata.clj:51)
	at metabase.sync.util$do_with_error_handling.invokeStatic(util.clj:149)
	at metabase.sync.util$do_with_error_handling.invoke(util.clj:142)
	at metabase.sync.util$do_with_error_handling.invokeStatic(util.clj:145)
	at metabase.sync.util$do_with_error_handling.invoke(util.clj:142)
	at clojure.core$partial$fn__5839.invoke(core.clj:2623)
	at metabase.driver$fn__22363.invokeStatic(driver.clj:537)
	at metabase.driver$fn__22363.invoke(driver.clj:537)
	at clojure.lang.MultiFn.invoke(MultiFn.java:239)
	at metabase.sync.util$sync_in_context$fn__44480.invoke(util.clj:138)
	at metabase.sync.util$with_db_logging_disabled$fn__44477.invoke(util.clj:129)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invokeStatic(util.clj:102)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invoke(util.clj:96)
	at metabase.sync.util$with_start_and_finish_logging$fn__44466.invoke(util.clj:114)
	at metabase.sync.util$with_sync_events$fn__44461.invoke(util.clj:88)
	at metabase.sync.util$with_duplicate_ops_prevented$fn__44452.invoke(util.clj:67)
	at metabase.sync.util$do_sync_operation.invokeStatic(util.clj:164)
	at metabase.sync.util$do_sync_operation.invoke(util.clj:161)
	at metabase.sync.sync_metadata$fn__54138$sync_db_metadata_BANG___54143$fn__54144.invoke(sync_metadata.clj:50)
	at metabase.sync.sync_metadata$fn__54138$sync_db_metadata_BANG___54143.invoke(sync_metadata.clj:47)
	at metabase.api.database$fn__54843$fn__54845.invoke(database.clj:590)
	at clojure.core$binding_conveyor_fn$fn__5754.invoke(core.clj:2030)
	at clojure.lang.AFn.call(AFn.java:18)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
07-06 23:30:35 INFO sync.util :: FINISHED: step 'sync-fields' for sqlite Database 5 'SQLite Food Trucks' (167.9 ms)
```

**Information about your Metabase Installation:**
Metabase 0.35.4

**Severity**
P1, since it can be very difficult to debug - specially, when people don't have access to backend.

**Comment by robdaemon** [comment]
This is a front-end specific issue, the UI never looks to see if `ev.exception` is set when rendering the logger messages. https://github.com/metabase/metabase/blob/master/frontend/src/metabase/admin/tasks/containers/Logs.jsx#L113-L117

**Comment by paulrosenzweig** [comment]
Fixed by #13501
